package.json

```diff
@@ -9,11 +9,24 @@
     "type": "git",
     "url": "https://github.com/pioneers/dawn.git"
   },
+  "prettier": {
+    "semi": true,
+    "singleQuote": true,
+    "printWidth": 140,
+    "arrowParens": "always",
+    "parser": "typescript",
+    "bracketSpacing": true
+  },
   "main": "build/main.js",
   "scripts": {
     "start": "electron build/main.js",
     "build": "tsc --build",
-    "lint": "eslint .",
+    "lint-actions": "eslint renderer/actions/**/*.ts",
+    "lint-reducers": "eslint renderer/reducers/**/*.ts",
+    "lint-types": "eslint renderer/types/**/*.ts",
+    "lint-actions-fix": "eslint renderer/actions/**/*.ts --fix",
+    "lint-reducers-fix": "eslint renderer/reducers/**/*.ts --fix",
+    "lint-types-fix": "eslint renderer/types/**/*.ts --fix",
```

Comment:
@HaileyJang Note here - These are temporary commands for linting in specific areas of the code like the actions, reducers, and types folders. Later on, we'll have 1 command that would lint the whole codebase.